formatTest/typeCheckedTests/expected_output/attributes.re

```diff
@@ -393,7 +392,7 @@ external render : (reactElement, element) => unit =
 external createCompositeElementInternalHack :
   (
     reactClass,
-    Js.t({.. reasonProps : 'props}),
+    Js.t({.. reasonProps: 'props}),
```

Comment:
But for record/object keys, it does make sense to have the colon be tightly bound. I'm leaning towards always printing the colon with spaces around it when it is a type annotation, and like `key: type` when it is a record/object label/type.